src/Cli/dotnet/ToolPackage/ToolPackageDownloader.cs

```diff
@@ -190,6 +197,52 @@ internal class ToolPackageDownloader : IToolPackageDownloader
                 });
         }
 
+        private static void IsRuntimeConfigCompatible(
+            ToolPackageInstance toolPackageInstance,
+            PackageId packageId,
+            bool isGlobalTool
+            )
+        {
+            var executableFilePath = toolPackageInstance.Commands[0].Executable;
```

Comment:
Fun fact: no, and it never was. I looked at [the commit that introduced IToolPackage and ToolPackageInstance](https://github.com/dotnet/sdk/commit/aab9af71b881b442aaa86182aa469ae199fb002c), and from the get go, ToolPackageInstance had a private field that was only set in one place, and it was set to a list with either 0 or 1 element. It could never have multiple. I'm going to do a fairly sizable refactor of that to avoid this confusion in the future.

Reply:
https://github.com/dotnet/sdk/pull/42866

Reply:
https://github.com/dotnet/sdk/pull/42866

Reply:
Now that @Forgind's change has merged, we probably need to look into this.

Reply:
Ah actually looks like we don't need to worry about this for now.